### PR TITLE
Submit csv button Functionality

### PR DESCRIPTION
**Pull request by carthy27**
I needed to wrap the function with the DomContentLoaded as it was loading the script before the dom was fully loaded. 

Added an eventlistener for the on submit button click. 

Added a parse csv to load modules 

Function adds to module and it is displayed on the frontend when submitted.

### CODE REVIEW

File: OBlock/data/modules.csv
Comment (Bourkupine) [inline OBlock/data/modules.csv:1]: we should probably test with different max capacities

File: OBlock/src/frontend.js [hunk 1]
```diff
@@ -48,4 +48,59 @@ function populateModuleDropdown() {
...
+
+    const reader = new FileReader();
+
+    reader.onload = function(e) {
+        const csvData = e.target.result;
+        parseCsvAndLoadModules(csvData);
+    };
+
+    reader.onerror = function() {
+        alert("An error occurred while reading the file.");
```
Comment (Bourkupine) [inline OBlock/src/frontend.js:72]: might be worth throwing a return here since its an error

**Review by Bourkupine — CHANGES_REQUESTED** [review]
Looks good bar the one comment about returning early when erroring

**Review by Bourkupine — APPROVED** [review]
Looks good

**Review by oOoOoOo0o — APPROVED** [review]

**Pull request merged**